### ISSUE TITLE
Loss function too hard to implement with Keras?

### ISSUE THREAD

**Issue by alexanderkoumis** [pradeeplam/Anime-Sketch-Coloring-with-Swish-Gated-Residual-UNet#1]
![image](https://user-images.githubusercontent.com/5108856/54313345-60071a80-4596-11e9-8342-178c7dd8d8dd.png)
